micro-rdk/src/native/entry.rs

```diff
@@ -100,7 +100,7 @@ pub async fn serve_web_inner(
         exec.clone(),
     ));
 
-    let mut srv = ViamServerBuilder::new(mdns, cloned_exec, client_connector, app_config)
+    let mut srv = ViamServerBuilder::new(mdns, cloned_exec, client_connector, app_config, 3)
```

Comment:
Should `native` be permitted more than 3? Or do you want to keep it the same with the idea that it should be a fairly faithful reproduction of how an ESP32 micro-rdk behaves?

Reply:
same comment as above